src/puppetlabs/puppetdb/scf/migrate.clj

```diff
@@ -727,7 +727,7 @@
              :values facts
              :timestamp timestamp
              :environment environment
-             :producer_timestamp nil}
+             :producer_timestamp timestamp}
```

Comment:
didn't read far enough. We have another migration down at the bottom that will handle anyone who already made it past this one. Should work fine.